databricks/koalas/frame.py

```diff
@@ -7964,13 +7960,15 @@ def info(
         ...           encoding="utf-8") as f:
         ...     _ = f.write(s)
         >>> with open('%s/info.txt' % path) as f:
-        ...     f.readlines()  # doctest: +NORMALIZE_WHITESPACE, +ELLIPSIS
-        [...databricks.koalas.frame.DataFrame...,
+        ...     f.readlines()  # doctest: +SKIP
```

Comment:
skip this for supporting pandas<1.0.0 & pandas>=1.0.0 both since their behavior are different depend on pandas version.

Reply:
Is the updated one from `pandas>=1.0`?

Reply:
@ueshin Not explicitly mentioned in the official 1.0.0 documentation, but it seems like that because in pandas 0.25.3 (which is the latest version before 1.0.0) API reference,

![스크린샷 2020-02-07 오후 3 39 17](https://user-images.githubusercontent.com/44108233/74006838-3f229180-49c0-11ea-9a27-ef90f3147379.png)

and pandas 1.0.0 API reference.

![스크린샷 2020-02-07 오후 3 39 29](https://user-images.githubusercontent.com/44108233/74006876-5497bb80-49c0-11ea-9860-adcf58b28cf6.png)

Reply:
I see, skipping this is fine.
btw, we should rather update the previous examples skipped above?

Reply:
okay, then I'll come back here to change this later when we totally migrate to pandas 1.0.0 with dropping the lower versions in the future.